hack/util.sh

```diff
@@ -74,6 +74,7 @@ function configure_os_server {
 	openshift start \
 	--write-config=${SERVER_CONFIG_DIR} \
 	--create-certs=false \
+	--dns="tcp://${API_HOST}:53" \
```

Comment:
> What do you mean? The patch by Stephen worked when it was merged in the past, right?

Also, if it was put there it was wrong, why would you go to the trouble of building a config that doesn't do what you want?